### PR TITLE
Fix graph_apply for symlinked configs

### PR DESCRIPTION
**Pull request by rddimon**
The `connect_symlinks` does not read `.tf_wrapper` and add folders to run.
Add temporary fix for checking `.tf_wrapper`

### CODE REVIEW

File: terrawrap/utils/config.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
 DEFAULT_REGION = 'us-west-2'
 SSM_ENVVAR_CACHE = SSMParameterGroup(max_age=600)
+TF_WRAP_FILE = ".tf_wrapper"
+TF_WRAP_FILE_YML = ".tf_wrapper.yml"
```
Comment (aoskotsky-amplify) [inline terrawrap/utils/config.py:26]: I don't think we need to support this filename. I don't see any cases where we use `.tf_wrapper.yml` in our code

File: terrawrap/utils/config.py [hunk 2]
```diff
@@ -62,7 +64,7 @@ def find_wrapper_config_files(path: str) -> List[str]:
     for element in elements:
         cur_path = os.path.join(cur_path, element)
         for file in os.listdir(cur_path):
-            if file.endswith(".tf_wrapper") or file.endswith(".tf_wrapper.yml"):
+            if file.endswith(TF_WRAP_FILE) or file.endswith(TF_WRAP_FILE_YML):
```
Comment (aoskotsky-amplify) [inline terrawrap/utils/config.py:67]:
```suggestion
            if file.endswith(TF_WRAP_FILE):
```

File: terrawrap/utils/graph.py [hunk 3]
```diff
@@ -135,8 +136,17 @@ def connect_symlinks(graph: networkx.DiGraph, symlink_dict: Dict[str, Set[str]])
...
-            graph.add_edge(current, secondary_link)
+
+            # TODO: implement better way for symlinked tf configs with terraform config
+            try:
+                links_to_run = walk_without_graph_directory(secondary_link)
+                for link in links_to_run:
+                    graph.add_edge(current, link)
+            except ValueError as error:
+                print(error)
+                graph.add_edge(current, secondary_link)
```
Comment (aoskotsky-amplify) [inline terrawrap/utils/graph.py:149]: Why only add secondary_link if there is an exception?

File: terrawrap/utils/config.py (hunk 1 above)
Comment (rddimon) [reply terrawrap/utils/config.py]: Done

File: terrawrap/utils/config.py (hunk 2 above)
Comment (rddimon) [reply terrawrap/utils/config.py]: Done

File: terrawrap/utils/graph.py (hunk 3 above)
Comment (rddimon) [reply terrawrap/utils/graph.py:150]: Added custom exception for this line.
`graph.add_edge(current, secondary_link)` here to keep previous behavior

**Review by jconstance-amplify — APPROVED** [review]

**Pull request merged**